core/src/main/java/org/jdbi/v3/core/statement/BaseStatement.java

```diff
@@ -123,7 +123,7 @@ public final boolean equals(Object o) {
 
     @Override
     public final int hashCode() {
-        return super.hashCode() * 11;
+        return System.identityHashCode(this);
```

Comment:
if this just takes the default hashcode behavior, delete the method and let `Object` use its implementation?

Reply:
can do.

Reply:
checkstyle will complain if we override equals but not hashcode.

Reply:
hm, looks like similarly, can we just delete the equals method that is the same as object default?